packages/cozy-sharing/src/components/ShareModal.spec.js

```diff
@@ -35,17 +35,4 @@ describe('ShareModal component', () => {
 
     expect(root.getByText('ShareDialogOnlyByLink'))
   })
-
-  it('should render ShareDialogCozyToCozy if type is Notes and notes.sharing-cozy-to-cozy flag enabled', () => {
```

Comment:
ce test est toujours pertinent (en supprimant `and notes.sharing-cozy-to-cozy flag enabled`) non ?

Reply:
En effet, je l'ai remis 👍